### PR TITLE
Only return non-zero scores

### PR DESCRIPTION
**Pull request by delkopiso**

### CODE REVIEW

**Comment by delkopiso** [comment]
@abepetrillo review?

File: app/controllers/api/scores_controller.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class ScoresController < ApplicationController
     before_action :authenticate!
 
     def index
-      @scores = Score.where(time_span: params[:time_span]).includes(:user)
+      @scores = Score.where(time_span: params[:time_span]).where("points > ?", 0).includes(:user)
```
Comment (abepetrillo) [inline app/controllers/api/scores_controller.rb]: I think displaying 0 woudl gives those concerned sufficient motivation to get some points :smile:

Comment (delkopiso) [inline app/controllers/api/scores_controller.rb]: Or not appearing at all might be even better motivation

**Comment by abepetrillo** [comment]
:+1:

**Pull request merged**